lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java

```diff
@@ -111,13 +115,22 @@ void shutdown() {
     void closeLightyModule(final LightyModule lightyModule) {
         if (lightyModule != null) {
             final Stopwatch stopwatch = Stopwatch.createStarted();
+            LOG.info("Lighty module {} shutting down ...", lightyModule);
             try {
-                LOG.info("Lighty module {} shutting down ...", lightyModule);
-                lightyModule.shutdown();
+                final ListenableFuture<Boolean> future = lightyModule.shutdown();
+                Futures.addCallback(future, new FutureCallback<>() {
+                    @Override
+                    public void onSuccess(Boolean result) {
```

Comment:
We can get `true` (closed successfully) but also `false` (closed unsuccessfully).

Reply:
Added return value.